### DISCUSSION TITLE
(TS2604) No defined construct or call signatures using on antd components

### DISCUSSION THREAD

**Discussion by JuanICasareski** [preactjs/preact#3754] [Q&A]
I'm trying to use TS + Preact + Antd to build an app, but all antd components appear to not have a defined construct or call signature. I tried the exact same setup in React and everything worked.

![image](https://user-images.githubusercontent.com/66040481/193489688-a8f3d6f5-0eb0-4fa0-a4c1-ec21c74e9d4b.png)

[reproduction repo](https://github.com/JuanICasareski/preact-antd-ts2604)

**Comment by Sukikiroi** [comment]
Hello,
  seems to be to downgrade antd to v3, and then everything seems to work as expected.
![image](https://user-images.githubusercontent.com/47844266/195791418-0be8de6b-e410-45c3-8455-0ca5ff8fbeaf.png)
